docs/docs/quick-start/storybook.md

```diff
@@ -39,3 +39,15 @@ Follow these steps to generate Storybook previews for each pull request.
 That's it. Now, every time you open a pull request, Stoat will host the Storybook build and post a comment with a link to the preview. See an example in [this pull request](https://github.com/stoat-dev/examples/pull/1).
 
 ![screen shot](https://user-images.githubusercontent.com/1933157/204390272-50819944-71bf-4037-b63f-5514c5c04edd.png)
+
+:::tip
+
+You can preview the static hosting comment locally by installing our CLI and launching local mode (requires Node/NPM):
+```
+npm i -g stoat
+stoat local
+```
+
+See our [cli guide](../tutorials/cli) for more information.
+
+:::
```

Comment:
You can move this block of text to a partial, and import it into other pages:
https://docusaurus.io/docs/markdown-features/react#importing-markdown

Reply:
I should also use partials to dry the quick start docs.